cwms_radar_api/src/main/java/cwms/radar/data/dto/Pool.java

```diff
@@ -7,8 +7,7 @@
 import usace.cwms.db.dao.ifc.pool.PoolNameType;
 import usace.cwms.db.dao.ifc.pool.PoolType;
 
-public class Pool extends PoolType implements CwmsDTO
-{
+public class Pool extends PoolType implements CwmsDTOBase {
```

Comment:
Should Pools have an office?

Reply:
Aren't there CWMS defined pools and then district pools?

Reply:
That's what I thought, which would necessitate the office. since we really just use "CWMS" as a "shared office" of sorts.

Reply:
https://bitbucket.hecdev.net/projects/SC/repos/hec-monolith/browse/hec-monolith/src/main/java/hec/data/pool/PoolName.java#22

Our data object has an office id and AV_POOL has an office column

Reply:
Ah, it's not a totally separate DTO. Didn't even notice that when I was reviewing. I'll change that as part of this PR. Last thing we want here is a real dependency on monolith, the DTOs are as much for setting the XML/JSON parameters as much as anything.

Reply:
Oh I linked the monolith representation. The one linked here is from the cwms-db-dao dependency which isn't based on monolith. Still it's really only a dumb object used to translate between jOOQ and monolith

Reply:
NOTE: I decided against fixing this for this PR, work will still need to get done.